definitions/variable/industry/tag_plastic_origin.yaml

```diff
@@ -0,0 +1,6 @@
+# Adjusted from metal origin definitions according to ISO IWA 19:2017(E)
```

Comment:
This tag seems not to be used in the variable definitions.

Reply:
Ah there is a typo in `production.yaml`. The chemicals should use this tag instead of the metal origin one.